tsuru/installer/installer.go

```diff
@@ -169,15 +173,32 @@ func (i *Installer) restartDocker(coreMachines []*dockermachine.Machine) error {
 	return nil
 }
 
-func (i *Installer) waitTsuru(cluster ServiceCluster, compConf *ComponentsConfig) error {
-	fmt.Println("Waiting for Tsuru API to become responsive...")
+func waitRegistry(cluster ServiceCluster) error {
+	manager := cluster.GetManager()
+	registryURL := fmt.Sprintf("https://%s:5000", dm.GetPrivateIP(manager))
```

Comment:
it may be best to get the registry URL in tsuru config (compConf.Tsuru.Config) because the registry address and port are configurable.

Reply:
👍